roles/slurm-stats/README.md

```diff
@@ -0,0 +1,32 @@
+stackhpc.slurm_openstack_tools.slurm-stats
+=========
+
+Configures slurm-stats from https://github.com/stackhpc/slurm-openstack-tools.git.
```

Comment:
Maybe a nit, but could we add e.g. :

> ... which transforms `sacct` output into a form that is more amenable for importing into elasticsearch/loki.

I know we don't want to duplicate docs all over the place but equally having to chase thro 3 repos to find out what does what makes it a bit hard to use.

Reply:
Added your suggestion.